release/models/platform/openconfig-platform.yang

```diff
@@ -1142,9 +1172,15 @@ module openconfig-platform {
 
       container config {
         description
-          "Configuration data for controller card components. Note that disabling
-          power to the primary supervisor should be rejected, and the operator is
-          required to perform a switchover first.";
+          "Configuration data for controller card components.
+          A controller-card can be configured for persistent
+          powered-off mode using the config/power-admin-state leaf.
+          The reference path below defines rules for such a
+          configuration.";
+
+        reference
+          "Rules around power-off configuration in controller-cards:
+          https://github.com/openconfig/public/blob/master/doc/controller-card_poweroff.md";
```

Comment:
Nit:  I think this description and reference should be moved to the controller-card model file (since this file is for power control for any type of component).

This would be a good location to add it I think: 
https://github.com/openconfig/public/blob/master/release/models/platform/openconfig-platform-controller-card.yang#L58-L65

Reply:
done